torch/nn/modules/conv.py

```diff
@@ -79,7 +79,7 @@ def _conv_forward(self, input: Tensor, weight: Tensor, bias: Optional[Tensor]) -
     transposed: bool
     output_padding: tuple[int, ...]
     groups: int
-    padding_mode: str
+    padding_mode: Literal["valid", "same"]
```

Comment:
This isn't correct at all.... See:
https://github.com/pytorch/pytorch/blob/f465318628c8bafa394a95466edaf4708a81b920/torch/nn/modules/conv.py#L121

I would hoist that set out of it and use typing.get_args to convert it into a set perhaps...

Accoding to the docs, padding_mode can be:
`padding_mode ([str](https://docs.python.org/3/library/stdtypes.html#str), optional) – 'zeros', 'reflect', 'replicate' or 'circular'. Default: 'zeros'`